### ISSUE TITLE
Antibiotics Volume Usage corrected for seasonality

### ISSUE THREAD

**Issue by brianmackenna** [ebmdatalab/openprescribing#1037]
A request for a new measure came up when testing the All England dashboard. The user would like a measure corrected for seasonality for the  [antibiotic volume measure ](https://openprescribing.net/measure/ktt9_antibiotics/)

> So seasonal - difficult to tell what it is. Can you adjust for seasonality? I suppose you can but there might be some way of presenting better but then will be different to all others so might confuse people. 
>

**Comment by sebbacon** [comment]
I'd love to be able to provide this.

We could use something like the `decompose` function [used by Alex in this notebook](https://github.com/ebmdatalab/antibiotics_itsa/blob/master/Antibiotics%20ITSA.ipynb).

However, our current workflow does everything in-SQL, so it would require some kind of python-based post-processing.

It may be better to look at this once we've progressed some of our other work relating to rationalising measures and our data formats.

In the mean time it would be helpful if @alexwalkercebm has a chance to try applying the `decompose` method to the antibiotic measure in a notebook, and seeing if it produces useful outputs.

**Comment by alexwalkercebm** [comment]
Like this?
https://gist.github.com/alexwalkercebm/4f884ea5d3b7ef4e014d61c6918a52b7

I've just smoothed the deciles, along with the global calc_value, it looks reasonable to me. It takes a fraction of a second to do the smoothing.

I think the ideal way to present this on the site could be as a `smooth` button/switch on existing measures, rather than as separate measures. No idea how difficult to implement that would be.

**Comment by brianmackenna** [comment]
Are we waiting for the matrix store for implementing this? Possible one for next top 3? 

Blog is [here](https://ebmdatalab.net/smoothing-seasonality-your-feedback-needed/) on prototyping for users.

**Comment by sebbacon** [comment]
Yes, we should wait ideally, simply because otherwise we'll just have to do it again in a different way
